testing/python_lib/test_failscale.py

```diff
@@ -35,23 +36,50 @@ def __init__(self, *args, **kwargs):
 
     def test_stack_connectivity(self):
         """Test to build stack and check for connectivity"""
+
+        faux_args = []
+        for dev_index in range(1, self.devices):
+            for sw_index in range(1, self.switches + 1):
+                device_num = dev_index * self.switches + sw_index
+                switch = 't2sw%s' % sw_index
+                faux_args.append((switch, 100 + dev_index + 1, device_num))
```

Comment:
extract "100 + ..." to named variable so it's obvious what it is (device_port?)

Reply:
Changed